### DISCUSSION TITLE
Replace "dds" in Names and Identifiers with "opendds"

### DISCUSSION THREAD

**Discussion by iguessthislldo** [OpenDDS/OpenDDS#2160] [Ideas]
**(Posting here mainly to test discussions)**

Things like `DDS_VERSION` really should be `OPENDDS_VERSION`. I'd ideally like this to extend to the `dds` directory, renaming it to `opendds`.

**Comment by simpsont-oci** [comment]
> I'd ideally like this to extend to the dds directory, renaming it to opendds.

Or, if our directory names were to follow modules / namespaces, it could be opendds (implementations) alongside dds (interfaces).

**Comment by jwillemsen** [comment]
Yes, would be in favor of that, would make it much clearer when we use OpenDDS in our AXCIOMA code or a different vendor. 

Also update the MPC base projects to use opendds as prefix.

**Comment by iguessthislldo** [comment]
Deprecating `DDS` version macros in #2333
